_linter/markdown-linter-style.rb

```diff
@@ -0,0 +1,33 @@
+#  Include all rules by default
+all
+
+#  Rule Modifications
+#  --------------------------------
+## MD029 mod - tells linter not to fail if order list items don't start with 1
```

Comment:
OK, I added a more descriptive comment now that hopefully explains this modification better.